TrackParamFilterFilter/sections/specific.txt

```diff
@@ -0,0 +1,3 @@
+!
+! The rules for specific sites
```

Comment:
Since you are not indicating specifically which site, so, I think `! Rules for specific sites` sounds better.